UpdateCosmosMessageView/handler.ts

```diff
@@ -38,7 +38,7 @@ export const handle = (
         TE.fromPredicate(
           messageStatusWithFiscalCode =>
             messageStatusWithFiscalCode.status !==
-            MessageStatusValueEnum.PROCESSED,
+            NotRejectedMessageStatusValueEnum.PROCESSED,
```

Comment:
I'd rather open a new PR where update function-commons library and refactor old code, instead of having all changes in this one, so we can deploy them separately.

Reply:
You are right, i've opened [this](https://github.com/pagopa/io-functions-messages-cqrs/pull/22)